lua/terrortown/menus/gamemode/changelog.lua

```diff
@@ -30,3 +33,23 @@ end
 function CLGAMEMODEMENU:GetSubmenus()
 	return virtualSubmenus
 end
+
+-- overwrite and return true to enable a searchbar
+function CLGAMEMODEMENU:HasSearchbar()
+	return true
+end
+
+-- overwrite and do a custom search inside title, date and content
+function CLGAMEMODEMENU:SearchFunction(submenuClass, searchText)
+	local txt = stringLower(searchText)
+	local change = submenuClass.change
+
+	if stringFind(stringLower(TryT(submenuClass.title)), txt) then return true end
+
+	if change.date > 0
+	and stringFind(stringLower(os.date("%Y/%m/%d", change.date)), txt) then return true end
```

Comment:
```suggestion
	if change.date > 0
		and stringFind(stringLower(os.date("%Y/%m/%d", change.date)), txt)
	then
		return true
	end
```

If you want to lines, then do it like this. Also returns with a value should never be in line.